src/rtssh.erl

```diff
@@ -250,10 +250,13 @@ stop(Node) ->
     run_riak(Node, "stop"),
     ok.
 
-upgrade(Node, NewVersion) ->
-    upgrade(Node, NewVersion, same).
+upgrade(Node, NewVersion, UpgradeCallback) when is_function(UpgradeCallback) ->
+    upgrade(Node, NewVersion, same, UpgradeCallback).
 
-upgrade(Node, NewVersion, Config) ->
+%% upgrade callback unsupported for this driver until there is a need.
+%% c.f., rtdev:upgrade/4
+upgrade(Node, NewVersion, Config, _UpgradeCallback) ->
+    lager:warning("Upgrade callback not supported in the rtssh riak test harness!"),
```

Comment:
So the trouble I see with this is that we're going to see this warning printed *every* time an upgrade is run using rtssh, regardless of whether the test specifies an upgrade callback or not. (We will still get `fun no_op/1` passed in here.)